src/components/Experience/Experience.tsx

```diff
@@ -21,7 +21,7 @@ const Experience = (): JSX.Element => {
 					<p>
 						{work.startDate} - {work.endDate}
 					</p>
-					<p>{workDescription}</p>
+					<>{workDescription}</>
```

Comment:
minor warning we were not aware of: `ul` property should not be in a `p` property, fixed with this update